doc/src/vpr/command_line_usage.rst

```diff
@@ -1517,6 +1517,21 @@ VPR uses a negotiated congestion algorithm (based on Pathfinder) to perform rout
     * `swns` - setup Worst Negative Slack (sWNS) [ns]
     * `stns` - Setup Total Negative Slack (sTNS) [ns]
 
+
+.. option:: --generate_net_timing_report {on | off}
+
+    Generates a net timing report for each net in the design. For each net, the timing information written in the following format:
+
+    .. code-block:: none
+
+        netname : Fanout : 
+        (bounding_box_xmin,bounding_box_ymin,bounding_box_layer_min),(bounding_box_xmax,bounding_box_ymax,bounding_box_layer_max) : 
+        source_instance <slack_on source pin> : 
+        <load pin name1> <slack on load pin name1> <net delay for this net> : 
+        <load pin name2> <slack on load pin name2> <net delay for this net> : ...
```

Comment:
Any reason for this particular format?  A .csv file would be easier to load into a spreadsheet an examine.

Reply:
Actually I would leave it compatible with Synplify (that is useful) but just comment that is the reason for this format.

Reply:
I initially thought the file would be directly consumed by Synplify, but apparently the synthesis team will be doing post-processing on it anyway, so it can be consumed by Synplify.